pages/matches/[id].js

```diff
@@ -15,7 +17,32 @@ const currentMatch = matches.find((match) => (
 ))
 
 //filter the bars, which contain the id of the current match
-const currentBars = currentMatch ? (bars.filter((bar) => (bar.matches.includes(currentMatch.id)))) : null
+const currentBars =(bars.filter((bar) => (bar.matches.includes(currentMatch?.id))))
+
+const [updatedBars, setUpdatedBars] = useState(currentBars)
+
+useEffect(() => {
+    setUpdatedBars(bars.filter((bar) => bar.matches.includes(currentMatch?.id)));
+  }, [bars, currentMatch]);
+
+
+// create function to handle the submit event
```

Comment:
For this comment I would say that the naming of your function "handleSubmit" has a very good name that describes it purpose without your further explanation needed. Therefore, I would remove this comment, but that is up to you to decide.

Reply:
removed :D